### PR TITLE
Remove python3.3 support.

### LINKED ISSUE #16: Drop support for Python 3.3
Hello,

You should consider dropping python3.3 support. It fails in travis since the used version of ```wheel```no longer supports it. Thats why the build of the pull request of @n0sk and me fails.

### PR DESCRIPTION
**Pull request by ghost**
Remove python3.3 since it is no longer support by major packages like wheel.

Fixes #16 
Blocks https://github.com/PaesslerAG/django-admin-caching/pull/15 (as seen here https://travis-ci.org/PaesslerAG/django-admin-caching/jobs/392696178#L488)

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
+   py{27,34,35}-django18,
    py{27,34,35}-django1{9,10},
    py{27,34,36}-django111,
 
 [testenv]
 commands =
     pip install -e tests
     make docs lint test
-    django111: make verify-is-on-latest-django
+    django20: make verify-is-on-latest-django
```
Comment (kairichard) [inline tox.ini:12]: That is probably the change for #15?

Comment (kairichard) [reply tox.ini:12]: I would guess that this also breaks travis.

Comment (kairichard) [reply tox.ini:12]: My bad, it does not break.

**Comment by ghost** [comment]
Hello @kairichard,
Could you review the PRs if you have the time?

**Pull request merged**